gems/laa_multi_step_forms/app/controllers/steps/base_step_controller.rb

```diff
@@ -40,8 +37,6 @@ def update_and_advance(form_class, opts = {})
         # Validations will not be run when saving a draft
         @form_object.save!
         redirect_to opts[:after_commit_redirect_path] || nsm_after_commit_path(id: current_application.id)
-      elsif params.key?(:reload)
```

Comment:
I won't keep commenting on these throughout, but again - looks like you're accidentally undoing a bunch of work from previous tickets at the moment....